### DISCUSSION TITLE
create private list

### DISCUSSION THREAD

**Discussion by mhk67** [community/community#12545] [Other Feature Feedback, Questions, & Ideas]
hi, if i can create private list is better than now 
thanks

**Comment by RicardoPoleo** [comment]
I would rather say that Lists, just like Repositories, should have a visibility setting that allows either to make them Visible/Invisible or Public/Private.

- Maybe you want a list of boilerplates/tools you want to use but not necessarily share
- Maybe your public lists can, later on, be followed by other members 

The scope of features that this would enable is pretty wide
